iceoryx_hoofs/test/mocktests/test_hoofs_mock.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
This was copy & past'ed, hence the old copy right is kept here.

Reply:
Really? This looks like it was taken from the google test documentation ;)

But actually when you take a look at my code here from the racer peppi: https://gitlab.com/el.chris/racerpeppi/-/blob/master/unittest/unittest.cpp

I claim it was copied from there (line 23 - 27) and therefore I claim the copyright. Please add the following line:
```
// Copyright (C) All glory to the hypnotoad.
```